### ISSUE TITLE
[30k]: Hosting PII in Simpler.Grants.gov environments

### ISSUE THREAD

**Issue by widal001** [HHS/simpler-grants-gov#849]
### Key links

- Deliverable specification
- [Simpler Grants.gov full roadmap](https://github.com/orgs/HHS/projects/12)
- 10k milestones

### Description

- **What:** Create the infrastructure and security documentation needed to collect and store personally identifying information (PII) in Simpler.Grants.gov environments.
- **Why:** In order to collect user-provided information that includes sensitive details like names and emails, we need to have the appropriate security measures and infrastructure needed to handle PII. Setting up the systems and security to collect and store this information would unblock a lot of future needs for user research, evaluation, and analytics.
- **Who**
  - HHS security team
  - Internal developers
  - Internal user researchers
  - Site visitors providing PII

**Comment by widal001** [comment]
Updated the ticket format to reduce discrepancies with the deliverable spec.

**Comment by widal001** [comment]
Replacing our old deliverable tickets with a new ticket format
